db/migrate/20160310132620_move_loc_id_to_own_model.rb

```diff
@@ -0,0 +1,53 @@
+class MoveLocIdToOwnModel < MigrationWithData
+  def up
```

Comment:
As soon as Hets implements that symbols and sentences are not allowed to have the same name, the corresponding version will fail and the Ontohub-database will not have the problem of duplicates.